### ISSUE TITLE
[py3k] ImportError: cannot import name Signal

### ISSUE THREAD

**Issue by queertypes** [davidcrawford/chronic#2]
``` pycon
Python 3.3.2 (default, May 29 2013, 11:42:12) 
[GCC 4.8.0] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import chronic
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
  File "/home/alejandro/.venv/q3/lib/python3.3/site-packages/chronic.py", line 38, in <module>
    from signals import Signal
  File "/home/alejandro/.venv/q3/lib/python3.3/site-packages/signals/__init__.py", line 4, in <module>
    from signals import Signal
ImportError: cannot import name Signal
```

This seems to be related to the way imports work in Python 3. The **init**.py is trying to pick up `Signal` from `signals/signals.py`.

**Comment by davidcrawford** [comment]
Thanks for the report, I'll look into it.

**Comment by davidcrawford** [comment]
Fixed with d98e5be
